migrations/20160127110439-account-fingerprint.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Have we already removed migrations? If not, it would be best to do that in one go since it looks like there are some other things that would have to change